stencil/src/components/resource-links/resource-links.scss

```diff
@@ -0,0 +1,20 @@
+pulumi-resource-links{
+    .container {
+        margin-bottom: 24px;
+    }
+    .heading {
+        color: #2e3872;
+        font-weight: 600;
```

Comment:
Are we able to use tailwind styles in stencil components here? When I use @apply they dont seem to take effect. I can inline the tailwind classes in the mark up if that is the case.